src/IRCNetwork.cpp

```diff
@@ -369,6 +369,21 @@ struct TOption {
     void (CIRCNetwork::*pSetter)(T);
 };
 
+bool CIRCNetwork::ModuleInConfig(CConfig* pConfig, const CString& sModule) const {
+    VCString vsList;
+    bool bFound;
```

Comment:
This is undefined behavior without `= false`
Compiler is free to optimize this whole function to `return true;`